zest/releaser/pypi.py

```diff
@@ -128,43 +136,46 @@ def fix_config(self):
         with open(self.config_filename) as config_file:
             print("".join(config_file.readlines()))
 
-    def python_file_with_version(self):
-        """Return Python filename with ``__version__`` marker, if configured.
-
-        Enable this by adding a ``python-file-with-version`` option::
-
-            [zest.releaser]
-            python-file-with-version = reinout/maurits.py
-
-        Return None when nothing has been configured.
+    def zest_releaser_config(self):
+        if not self.config:
+            return None
 
-        """
-        default = None
-        if self.config is None:
-            return default
         try:
-            result = self._get_text(
-                "zest.releaser", "python-file-with-version", default=default
-            )
-        except (NoSectionError, NoOptionError, ValueError):
-            return default
+            result = dict(self.config["zest.releaser"].items())
+        except KeyError:
+            return None
+
+        boolean_keys = [
+            "release",
+            "create-wheel",
+            "no-input",
+            "register",
+            "push-changes",
+            "less-zeroes",
+            "tag-signing",
+            "run-pre-commit",
+        ]
+        for key, value in result.items():
+            if key in boolean_keys:
+                result[key] = string_to_bool(value)
+            if key in ["version-levels"]:
```

Comment:
Can you turn this into a separate `integer_keys` variable to mimic `boolean_keys`?